models/migrations/v184.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"fmt"
+
+	"code.gitea.io/gitea/modules/setting"
+	"xorm.io/xorm"
```

Comment:
Tiniest nit, could you put empty line between these imports

Reply:
done.